data/phones/list_phones.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+"""This script prints a tally of the phones/phonemes of a WikiPron TSV file.
+
+For each phone/phoneme, this script prints:
+- the phone/phoneme
+- the number of words that have this phone/phoneme
+- a few example word-pronunciation pairs for this phone/phoneme
+"""
+
+import argparse
+import collections
+import random
+
+from typing import Dict, List, Set
+
+
+def _get_cli_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("filepath", help="Path to TSV scraped by WikiPron")
+    args = parser.parse_args()
+    return args
+
+
+def _count_phones(filepath: str) -> Dict[str, Set[str]]:
+    # phone_to_examples as Dict[str, Set[str]] is the most straightforward
```

Comment:
What about changing this to a [PEP 257](https://www.python.org/dev/peps/pep-0257/) docstring instead?